python/ray/dag/tests/experimental/test_accelerated_dag.py

```diff
@@ -461,6 +461,75 @@ def test_chain_dag(ray_start_regular, num_actors):
     compiled_dag.teardown()
 
 
+def test_execution_timeout(ray_start_regular):
+    a = Actor.remote(0)
+    with InputNode() as inp:
+        dag = a.inc.bind(inp)
+
+    compiled_dag = dag.experimental_compile(execution_timeout=3)
+    refs = []
+    timed_out = False
+    for i in range(5):
+        try:
+            ref = compiled_dag.execute(1)
+            # Hold the refs to avoid get() being called on the ref
+            # when it goes out of scope
```

Comment:
Could you clarify this?

Reply:
I guess Rui is referring to the `__del__` method for `CompiledDAGRef`.

https://github.com/ray-project/ray/blob/755a49bbc2345650ff90d4706d9c69f87b8a77bd/python/ray/experimental/compiled_dag_ref.py#L75-L78

Reply:
Kai-Hsun is right. Made the comment more explicit.